pod/perlop.pod

```diff
@@ -1517,11 +1519,22 @@ takes higher precedence.
 
 Then again, you could always use parentheses.
 
-Binary C<"xor"> returns the exclusive-OR of the two surrounding expressions.
-It cannot short-circuit (of course).
-
 There is no low precedence operator for defined-OR.
 
+=head2 Logical Exclusive Or
+X<operator, logical, xor> X<operator, logical, exclusive or> X<xor>
+
+Binary C<"xor"> returns the logical disjunction of the two surrounding
+expressions.  That means it returns 1 if either, but not both, are true.
+It's equivalent to C<^> except for it having very low precedence.  It
+tends to be used to verify that two mutually-exclusive conditions are
+actually mutually exclusive.  For example, in Perl's test suite, we
+might want to test that a regular expression pattern can't both match
+and not match, for otherwise it would be a bug in our pattern matching
+code.
+
+ ($a =~ /$pat/ xor $a !~ /$pat/) or die;
```

Comment:
with empty regex `//` to be same as defined-or operator `//` it will be nice to not promote operator-less regular expressions:
```
$a =~ $pat
$a =~ qr /$pat/
```

Reply:
Thanks; now fixed